erizo/src/erizo/rtp/RtpHeaders.h

```diff
@@ -420,9 +420,18 @@ namespace erizo{
       }
       inline uint32_t getOctetsSent(){
         return ntohl(report.senderReport.octetssent);
-      }      
+      }
       inline uint64_t getNtpTimestamp(){
-       return (((uint64_t)htonl(report.senderReport.ntptimestamp)) << 32) + htonl(report.senderReport.ntptimestamp >> 32);
+        return (((uint64_t)htonl(report.senderReport.ntptimestamp)) << 32) + htonl(report.senderReport.ntptimestamp >> 32);
```

Comment:
@lodoyun could you please check this? This code was already committed in the current master branch.